pillar/top.sls

```diff
@@ -6,5 +6,6 @@ base:
     - ip_addrs
     - fqdn
     - kubelet
+    - availability_zone
```

Comment:
I think the way it's setup, there was always be values.. However, wrapping it in a `{% if node_labels %} .. {% endif %}` is harmless and prevents it breaking if `--node-labels=` would otherwise cause an issue..